Mage/src/main/java/mage/filter/predicate/permanent/PermanentInListPredicate.java

```diff
@@ -20,6 +20,6 @@ public PermanentInListPredicate(List<Permanent> permanents) {
 
     @Override
     public boolean apply(Permanent input, Game game) {
-        return permanents.contains(input);
+        return (permanents != null && permanents.contains(input));
```

Comment:
No needs in null protection for class's required fields, it will hide real error in the future (wrong predicate usage with broken param). All critical errors and wrong code usages should be reported as soon as possible (exceptions for server logs and user's feedback about broken card).

Reply:
Ah, ok then, I should null check before making the Predicate in all scenarios then?

Reply:
Yes, if you use PermanentInListPredicate or the new one then must calls it with correct data.